CMakeLists.txt

```diff
@@ -159,20 +159,45 @@ if (WIN32)
     add_definitions(-DNOMINMAX -DWIN32_LEAN_AND_MEAN)
 endif()
 
-# Dependencies
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
+    set(OSG_PLUGINS_DIR ${OSG_PLUGINS_DIR})
+    add_definitions (-DOSG_PLUGINS_DIR)
+    set(OPENGL_ES TRUE CACHE BOOL "enable opengl es support for android" FORCE)
+endif (ANDROID)
 
-set(DESIRED_QT_VERSION 4 CACHE STRING "The QT version OpenMW should use (4 or 5)")
-message(STATUS "Using Qt${DESIRED_QT_VERSION}")
+option(OPENGL_ES "enable opengl es support" FALSE )
 
-if (DESIRED_QT_VERSION MATCHES 4)
-    find_package(Qt4 REQUIRED COMPONENTS QtCore QtGui QtNetwork QtOpenGL)
+option(USE_QT "Use Qt in building" TRUE )
+
+if (NOT BUILD_LAUNCHER AND NOT BUILD_OPENCS AND NOT BUILD_WIZARD)
+    set(USE_QT FALSE CACHE BOOL "disable Qt" FORCE )
 else()
-    find_package(Qt5Widgets REQUIRED)
-    find_package(Qt5Core REQUIRED)
-    find_package(Qt5Network REQUIRED)
-    find_package(Qt5OpenGL REQUIRED)
+    set(USE_QT TRUE CACHE BOOL "enable Qt" FORCE )
+endif()
+
+add_definitions (-DUSE_QT)
+ 
+if (OPENGL_ES)
+    find_package(OpenGLES)
```

Comment:
You want to add a REQUIRED so the cmake script will complain when the OpenGL ES library is not found.
<pre>
find_package(OpenGLES REQUIRED)
</pre>

Also, it might be better to move the find_package call to the file where the ```${OPENGLES_gl_LIBRARY}``` is actually used, i.e. in components/CMakeLists.txt next to ```find_package(OpenGL REQUIRED)```.